flink-connectors/flink-connector-hbase-base/src/test/java/org/apache/flink/connector/hbase/util/HBaseConfigLoadingTest.java

```diff
@@ -64,13 +64,13 @@ public void loadFromEnvVariables() throws Exception {
         final String k4 = "which way?";
         final String v4 = "south, always south...";
 
-        final File hbaseConfDir = tempFolder.newFolder();
+        final File hbaseConfDir = tmpDir.toFile();
 
-        final File hbaseHome = tempFolder.newFolder();
+        final File hbaseHome = Files.createTempDirectory(tmpDir, "junit_hbaseHome_").toFile();
```

Comment:
fyi: There's also `TempDirUtils.newFolder(path)` which might be less verbose. But your approach works as well.